processors/base/writers/publication.py

```diff
@@ -43,7 +43,7 @@ def write_publication(conn, publication, source_id):
         create = True
 
     # Write object only for high priority source
-    if create or source_id in ['pubmed']:
+    if create or source_id in ['pubmed', 'hra']:
```

Comment:
@vitorbaptista I will assign you to this PR, can you do the modification? I'm not sure if I should remove all the conditions or just the `source_id` ones 😅